wordpress-mobile/WordPress-Aztec-iOS
# More: Public API + Format Bar Support

### Details:
In this PR we're implementing a public API to support the insertion of the More Attachment.

Please, note that (since we're updating the exact same spot), we've patched the `HR` tag "diffing" snippet, since HR tags can be only inserted, not removed, by means of string diffing.

Closes #361

### Testing:
- Verify that all of the unit tests pass
- Verify that pressing the `More` format bar button effectively inserts the more section
- Verify the generated HTML
- Verify that inserting `` in html mode (case insensitive) renders as an attachment, in rich mode

Needs Review: @diegoreymendez + @SergioEstevao
Thanks in advance!!
